### PR TITLE
📜 Revise the `docs` to remain in sync with the current changes

### LINKED ISSUE #148: 📜 Revise the `docs` to remain in sync with the current changes
Revise the `docs` to add documentation covering the new option and feature provided by the PRs #146, #77 and #133.

### PR DESCRIPTION
**Pull request by neon-mmd**
## What does this PR do?

This PR updates the `docs` for the project by updating the names of the previous config options and adding and explaining new config option proposed in PR #145 and also updates the explanation of the colorscheme option.

## Why is this change important?

This change is essential as it ensures that the `docs` remain in sync with the current changes.

## Author's checklist

- [x] update the name of `redis_connection_url` config option.
- [x] add a new config option `upstream_search_engines` with the explanation.
- [x] update the explanation of the config option `colorscheme`.  
- [x] bump the app version 

## Related issues

Closes #148

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -36,14 +37,17 @@ Some of the configuration options provided in the file are stated below. These a
 > By Default we provide 9 colorschemes to choose from these are:
```
Comment (xffxff) [inline docs/configuration.md]: We currently provide 12 themes

**Review by xffxff — DISMISSED** [review]
Looks good to me, just one small issue needs to be fixed

**Comment by neon-mmd** [comment]
If you don't mind can you kindly commit this suggestion from your side if possible 🙂 because I am abroad for 9-10 days so I might not be able to do so.

**Comment by xffxff** [comment]
> If you don't mind can you kindly commit this suggestion from your side if possible 🙂 because I am abroad for 9-10 days so I might not be able to do so.

Sure, no problem. Wishing you a pleasant journey.

File: docs/configuration.md (hunk 1 above)
Comment (xffxff) [inline docs/configuration.md]:
```suggestion
> By Default we provide 12 colorschemes to choose from these are:
```

**Review by xffxff — APPROVED** [review]

**Pull request merged**